models/Drone.model.js

```diff
@@ -1 +1,14 @@
-// Iteration #1
\ No newline at end of file
+// Iteration #1
+ 
+const {Schema, model} = require("mongoose")
+
+const DroneSchema = new Schema({
```

Comment:
Maybe add more validation like make the name required and the Number properties control also the `min: 0` value